cardano-cli/test/cardano-cli-golden/files/golden/help/conway_transaction.cli

```diff
@@ -24,6 +25,9 @@ Available commands:
                            Please note the order of some cmd options is crucial. If used incorrectly may produce undesired tx body. See nested [] notation above for details.[0;22;23;24m
   build                    Build a balanced transaction (automatically calculates fees)
 
+                           Please note the order[93;22;23;24m of some cmd options is crucial. If used incorrectly may produce undesired tx body. See nested [] notation above for details.[0;22;23;24m
```

Comment:
where those escape characters come from?

Reply:
`opt-parse-applicative`. It's the colouring of the text.